vud.theme.inc

```diff
@@ -114,10 +128,10 @@ function vud_pseudo_theming($vote_entity, $template_type, $plugin, &$variables)
     global $theme_key;
     $themes = list_themes();
     $current_theme = $themes[$theme_key];
-    $paths = array(
+    $paths = [
```

Comment:
You are changing array syntax in many places in this file, I do not see how that is related to the widgets change. Let's revert them if there is not a reason.